lib/engaging_networks_rest/client.rb

```diff
@@ -11,18 +12,25 @@ class Client
     def initialize(api_key:)
       @api_key = api_key
 
-      @connection = Faraday.new(url: "https://#{ENS_DOMAIN}")
+      @connection = Faraday.new(url: "https://#{ENS_DOMAIN}") do |conn|
+        conn.request :json
+        conn.response :json, :content_type => /\bjson$/
+
+        conn.use Faraday::Response::Logger if ENV['DEBUG']
+        conn.use EngagingNetworksRest::Response::RaiseError
+
+        conn.adapter :net_http
```

Comment:
Why are we forcing net::http here? It's the least performant of the pluggable adapters that faraday supports.

Reply:
Faraday was displaying a warning message before when the adapter was not being explicitly set. I checked and this is what we do by default in other gems (i.e.: Action Kit)